pymeasure/instruments/signalrecovery/dsp7265.py

```diff
@@ -199,19 +219,12 @@ def __init__(self, resourceName, **kwargs):
             resourceName,
             "Signal Recovery DSP 7265",
             includeSCPI=False,
+            # Remove extra unicode character
+            # TODO: (find a way to) test if the line underneath indeed correctly replaces the reimplemented values method
+            preprocess_reply=lambda r: r.replace('\x00', ''),
```

Comment:
funny, I had another instrument model where some of the devices would return spurious \x00 bytes, do you know what causes this in your case?

Reply:
In principle this functionality is already being tested [here](https://github.com/pymeasure/pymeasure/blob/0670a8ab4174fdeaf498c92f1d81042b7f705de1/tests/instruments/test_instrument.py#L175-L192). I guess you can test with your physical device to confirm the bytes get swallowed?

Reply:
Well, my device (the one I used for testing this PR) doesn't do this. I left it there because originally, in this instrument the `values` method was reimplemented just for this purpose; maybe it also depends on the way you communicate with the system (GPIB vs. serial e.g.).